### PR TITLE
feat: Text Utilities added to Library

### PR DESCRIPTION
**Pull request by mdarif6**
feat: Text Utilities added to Library

### CODE REVIEW

**Review by Shah-Arsalan — APPROVED** [review]
code is good . nothing out of place . You can merge

File: components/text-utilities/text-utilities.css [hunk 1]
```diff
@@ -0,0 +1,50 @@
+h1,
+h2,
+h3,
+h4,
+h5,
+h6,
+.h1,
+.h2,
+.h3,
+.h4,
```
Comment (SJTGSHIVAM) [inline components/text-utilities/text-utilities.css:10]: I like the way you made it simple and clear.

File: document/text-utilities.html [hunk 2]
```diff
@@ -0,0 +1,261 @@
...
+            ></a>
+          </li>
+          <li class="pd-footer-icons social-icon">
+            <a class="link" href="https://twitter.com/mdarifdev"
+              ><i class="fab fa-twitter"></i
+            ></a>
+          </li>
+        </ul>
+        <div>
+          <small>&copy; Md Arif 2022 </small>
```
Comment (SJTGSHIVAM) [inline document/text-utilities.html:255]: I would definitely say this is nice usage of semantic tags

**Review by SJTGSHIVAM — APPROVED** [review]
It was a small change, overall everything looks good.

**Pull request merged**